src/main/scala/beam/sim/BeamSim.scala

```diff
@@ -195,8 +195,8 @@ class BeamSim @Inject()(private val actorSystem: ActorSystem,
 
   private def initVehicleActors(iterId: Option[String] = None) = {
     val actors = services.vehicles.map { case (vehicleId, matSimVehicle) =>
-      val props = BeamVehicleAgent.props(vehicleId, matSimVehicle, new Powertrain(BeamVehicle.energyPerUnitByType(matSimVehicle.getType.getId)))
-      val beamVehicle = actorSystem.actorOf(props, BeamVehicle.buildActorName(vehicleId, iterId))
+      val props = BeamVehicle.props(vehicleId, matSimVehicle, new Powertrain(BeamVehicle.energyPerUnitByType(matSimVehicle.getType.getId)))
```

Comment:
Yes, I left that chore for this week while we work toward getting this code to actually run...